### PR TITLE
{AKS} Retain value in network profile of mc object only when decorator is in update mode

### PR DESCRIPTION
**Pull request by MartinForReal**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[AKS] `az aks update`: Retain value in network profile in mc object only when decorator is in update mode

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py [hunk 1]
```diff
@@ -1716,6 +1716,7 @@ def _get_load_balancer_sku(self, enable_validation: bool = False) -> Union[str,
         load_balancer_sku = safe_lower(self.raw_param.get("load_balancer_sku", CONST_LOAD_BALANCER_SKU_STANDARD))
         # try to read the property value corresponding to the parameter from the `mc` object
         if (
+            load_balancer_sku is None and
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py:1719]: not possible to be None, its default value is `"standard"`

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py (hunk 1 above)
Comment (MartinForReal) [reply src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py]: removed. Thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**